app/src/main/java/com/github/swent/echo/compose/components/SearchMenuSheet.kt

```diff
@@ -52,129 +40,84 @@ fun SearchMenuSheet(
     pendingCallback: () -> Unit,
     confirmedCallback: () -> Unit,
     fullCallback: () -> Unit,
-    sortByCallback: (SortBy) -> Unit,
+    sortByCallback: (Int) -> Unit,
     resetFiltersCallback: () -> Unit,
-    timeFilterCallback: (Float, Float) -> Unit
+    timeFilterCallback: (Float, Float) -> Unit,
+    initialPage: Int,
+    mode: MapOrListMode
 ) {
-    // Search mode
-    val searchMode = remember { mutableStateOf(SearchMode.FILTERS) }
     // TagViewModel
     val tagViewModel: TagViewModel =
-        hiltViewModel<TagViewModel, TagViewModel.TagViewModelFactory>() { factory ->
+        hiltViewModel<TagViewModel, TagViewModel.TagViewModelFactory> { factory ->
             factory.create()
         }
 
     val sheetState =
-        rememberModalBottomSheetState(skipPartiallyExpanded = false) { value ->
+        rememberModalBottomSheetState(skipPartiallyExpanded = true) { value ->
             if (value == SheetValue.Expanded) {
                 onFullyExtended()
             }
             true
         }
+    val bottomSheetHeight = 500.dp
+    val paddingValues = 5.dp
+    val pageHeight = 385.dp
+    val paddingBottomResetFilters = 10.dp
     ModalBottomSheet(
-        modifier = Modifier.fillMaxWidth().fillMaxHeight(0.5f).testTag("search_menu_sheet"),
+        modifier = Modifier.fillMaxWidth().height(bottomSheetHeight).testTag("search_menu_sheet"),
```

Comment:
this is actually not recommended, fillMaxHeight(0.5f) does so that the sheet takes half the phone size, the way you did things makes it a fixed size. Unless you had a reason to do this I suggest keeping fillMaxHeight

Reply:
The reason is that each phone have a different size. If we have a fix size, we are guaranteed that all elements are always displayed the same and that there is no missing components on little phones